content/contributing/project-information/working-groups.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Working Groups"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Working groups are temporary community initiatives devoted to particularly complex, large, or important tasks. Some examples include:
```

Comment:
Nitpicky, but should probably be "and/or" instead of (exclusive) "or" :)

Reply:
Or is inclusive by default in programmer communities!